cli/tests/integration/fmt_tests.rs

```diff
@@ -129,7 +129,7 @@ fn fmt_ignore_unexplicit_files() {
 }
 
 itest!(fmt_check_tests_dir {
-  args: "fmt --check ./ --ignore=.test_coverage,fmt/fmt_with_config/",
+  args: "fmt --check ./ --ignore=.test_coverage,fmt/fmt_with_config/,test/markdown_windows.md",
```

Comment:
This is unfortunate, I opened https://github.com/denoland/deno/issues/12758 to fix this situation